tests/test_cli.py

```diff
@@ -219,7 +324,7 @@ def test_script_main():
     assert sorted(data_out.files) == sorted(data_out.files) == sorted(
         ['embedding', 'timestamps'])
     assert np.allclose(data_out['timestamps'], data_reg['timestamps'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
     assert np.allclose(data_out['embedding'], data_reg['embedding'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
```

Comment:
Just curious, why did we have to reduce the tolerance here?

Reply:
IIRC it was because of integration issues. I was able to get them all very close but not within the same tolerance as before. This indicated to me that it was more or less floating point issues that may have resulted rather than actual problem.

Reply:
👍